framework/src/base/Moose.C

```diff
@@ -75,6 +75,9 @@ addActionTypes(Syntax & syntax)
   /**** Register Actions ****/
   /**************************/
   registerMooseObjectTask("create_problem",               Problem,                false);
+  registerMooseObjectTask("create_problem_default",       Problem,                false);
+  registerMooseObjectTask("set_problem_parameters",       Problem,                false);
```

Comment:
No because it's associated with a MooseObjectAction and there's a sanity check to make sure that you can't add MooseObjects (even potentially add them because we don't actually know). If you try, MOOSE will complain that you aren't allowed to add Problems.

Reply:
Can this be solved by splitting the `CreateProblemAction`?